packages/babel-helper-transform-fixture-test-runner/src/index.ts

```diff
@@ -920,7 +920,14 @@ export function buildProcessTests(
                 : [test.binLoc];
 
             args = args.concat(opts.args);
-            const env = { ...process.env, FORCE_COLOR: "false", ...opts.env };
+            const env = {
+              ...process.env,
+              FORCE_COLOR: "false",
+              ...(parseInt(process.versions.node) >= 23 && {
+                NODE_OPTIONS: "--disable-warning=ExperimentalWarning",
```

Comment:
```suggestion
                NODE_OPTIONS: (process.env.NODE_OPTIONS || "") + " --disable-warning=ExperimentalWarning",
```
Otherwise the fixture test runner will overwrite any `NODE_OPTIONS` passed from command line, which could be very confusing for contributors.